op-node/rollup/driver/finalized.go

```diff
@@ -0,0 +1,22 @@
+package driver
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+type finalized struct {
+	derive.L1Fetcher
+}
+
+func NewFinalized(fetcher derive.L1Fetcher) *finalized {
+	return &finalized{L1Fetcher: fetcher}
+}
+
+func (f *finalized) L1BlockRefByNumber(ctx context.Context, num uint64) (eth.L1BlockRef, error) {
+	return f.L1BlockRefByLabel(ctx, eth.Finalized)
```

Comment:
I'm confused.. does the L1Fetcher already have a L1BlockRefByLabel? wasn't this added on the other PR?

Reply:
The `L1Fetcher` has it, `L1Blocks` doesn't have it.